pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -379,87 +377,15 @@ private String listTableConfigs(String tableName, @Nullable String tableTypeStr)
   @ManualAuthorization
   @Path("/tables/{tableName}")
   @ApiOperation(value = "Lists the table configs")
-  public String alterTableStateOrListTableConfig(
+  public String listTableConfig(
       @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
-      @ApiParam(value = "enable|disable|drop") @QueryParam("state") String stateStr,
       @ApiParam(value = "realtime|offline") @QueryParam("type") String tableTypeStr, @Context HttpHeaders httpHeaders,
       @Context Request request) {
-    try {
-      if (StringUtils.isBlank(stateStr)) {
```

Comment:
shall we still keep `@ApiParam(value = "enable|disable|drop") @QueryParam("state") String stateStr,` but instead throw exception of unsupported?

Reply:
We did something similar for instances in #10993 and removed the state parameter entirely. I think we can be a bit aggressive here since this shouldn't have been supported in GET in the first place.